### ISSUE TITLE
KVM / virt nodes ?

### ISSUE THREAD

**Issue by apatard** [ansible/ansible-zuul-jobs#654]
molecule-vagrant is needing support for virtualization and from time to time, the tests are failing due to missing support for  virtualization. The tests are failing like this:
```
centos-8 | Error while creating domain: Error saving the server: Call to virDomainDefineXML failed: invalid argument: could not find capabilities for domaintype=kvm
```
(ref: https://dashboard.zuul.ansible.com/t/ansible/build/426dd647e97141b9b87f49b6a4c4b364/log/job-output.txt)

Not only it makes the testsuite fail randomly, but it's also a waste of resources.
Is there a way to run tests only on hosts providing virtualization ?

**Comment by ssbarnea** [comment]
@pabelanger @goneri Can you please help/advise with this? This job really needs virtualization support, maybe we have specific nodesets we can mention to avoid failing randomly.

@apatard Were you able to find failures on other nodes than centos-8 or is specific to them? -- I see jobs running on other platforms at https://github.com/ansible-community/molecule-vagrant/blob/master/zuul.d/layout.yaml#L14-L17

**Comment by apatard** [comment]
No, it's not specific to centos. For instance, this old log on fedora: https://dashboard.zuul.ansible.com/t/ansible/build/8218cc176f9f401fbdbe16fe07b4f2a9/log/job-output.txt

**Comment by goneri** [comment]
I removed centos-8 from AWS for now, https://github.com/ansible-network/windmill-config/pull/740. I will see later how we can properly identify the nodes with the kvm capability and the others (EC2).

**Comment by goneri** [comment]
See: https://github.com/ansible-network/windmill-config/pull/743
